src/storage/txn/scheduler.rs

```diff
@@ -674,6 +679,15 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
     }
 }
 
+impl<E: Engine, L: LockManager, P: PdClient + 'static> Clone for Scheduler<E, L, P> {
```

Comment:
Why not derive `Clone`?

Reply:
Deriving Clone requires that the type parameters implement clone, but writing it ourselves we don't need to since they are wrapped in `Arc`s